packages/calypso-e2e/src/data-helper.ts

```diff
@@ -105,7 +105,7 @@ export function getCalypsoURL(
  * @returns {string[]} Username and password found in the secrets file for the given account type.
  * @throws {Error} If accountType does not correspond to a valid entry in the file.
  */
-export function getAccountCredential( accountType: string ): string[] {
+export function getAccountCredential( accountType: string ): [ string, string ] {
```

Comment:
I assume this change is to use the correct signature? I don't see anything else having changed in this method.

Reply:
This narrows the returned type so that it's spreadable, e.g.:

```js
await logIn( ...getAccountCredential() );
```